libs/@local/harpc/service/src/delegate.rs

```diff
@@ -0,0 +1,17 @@
+use harpc_tower::{body::Body, request::Request, response::Response};
+use harpc_wire_protocol::response::kind::ResponseKind;
+
+use crate::Service;
+
+pub trait ServiceDelegate<S, C> {
```

Comment:
Can we add a documentation for this trait please?

Reply:
see: [350295c](https://github.com/hashintel/hash/pull/5283/commits/350295c35004f4e27abc8ba6114adc8c42659e64)